cmd/client.cc

```diff
@@ -39,16 +39,12 @@ struct Delegate : public ITransport::TransportDelegate
     void on_connection_status(const TransportContextId& context_id, const TransportStatus status)
     {
         tcid = context_id;
-        std::stringstream s_log;
-        s_log << "Connection state change context: " << context_id << ", " << int(status);
-        logger.log(LogLevel::info, s_log.str());
+        logger->info << "Connection state change context: " << context_id << ", " << int(status) << std::flush;
```

Comment:
Is `std::flush` always needed?

Reply:
If using the streaming operator, yes. It flushes the buffer. It's like having `std::endl`.  But `std::endl` is used to insert new lines, so was not used.  If using the functions called (`logger.Log()`) or the macros like `LOGGER_DEBUG`, then std::flush is performed automatically.